guides/common/modules/proc_securing-the-dhcp-api.adoc

```diff
@@ -0,0 +1,35 @@
+[id="Securing_the_dhcpd_API_{context}"]
+= Securing the dhcpd API
+
```

Comment:
WTF is dhcpd API, how does it relate to a DHCP provider or SmartProxy and why should I care about securing it? :D

Reply:
I think this is a bit below:
> By default, the dhcpd API listens to any host.
> You can add an `omapi_key` to provide basic security.

Should this be phrased stronger? Like

```
By default, the dhcpd API listens to any host without access controls.
```

Reply:
I have applied your suggestion.